packages/backend/src/api/controllers/UserController.ts

```diff
@@ -553,9 +552,9 @@ function buildUserTransactions(
   collateralAsset?: CollateralAsset,
   assetDetailsMap?: AssetDetailsMap
 ): TransactionEntry[] {
+  const userTransactionHashes = userTransactions.map((t) => t.transactionHash)
   const sentEntries = sentTransactions
-    // Mined non-reverted transactions will be inside userTransactions
-    .filter((t) => t.mined === undefined || t.mined.reverted)
+    .filter((t) => !userTransactionHashes.includes(t.transactionHash))
```

Comment:
I think we should show "Reverted" transactions in user history. But maybe I misunderstand the purpose of this function here.

Reply:
revist after log frozen